### PR TITLE
Show destinations in tx details popup

### LINKED ISSUE #229: [Feature] Add recipient's address to transaction details in transaction history
This is consistent with the behavior of `show_transfers` in `monero-wallet-cli`. In addition, this makes it more convenient for users to verify their payment.

### PR DESCRIPTION
**Pull request by Jaqueeee**
Fixes #229

Also added Transfer class for later use.

### CODE REVIEW

File: src/libwalletqt/Transfer.h [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+class Transfer : public QObject
+{
+    Q_OBJECT
+    Q_PROPERTY(quint64 amount READ amount)
+    Q_PROPERTY(QString address READ address)
+private:
+    explicit Transfer(uint64_t _amount, const QString &_address,  QObject *parent = 0): m_amount(_amount), m_address(_address), QObject(parent) {};
+private:
+    friend class TransactionInfo;
+    qint64 m_amount;
```
Comment (moneromooo-monero) [inline src/libwalletqt/Transfer.h]: Is there a good reason why this is signed ?

File: src/libwalletqt/Transfer.h [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    QString m_address;
+
+public:
+    qint64 amount(){ return m_amount; }
```
Comment (moneromooo-monero) [inline src/libwalletqt/Transfer.h]: Same about signed, also const

File: src/libwalletqt/TransactionInfo.cpp [hunk 3]
```diff
@@ -71,6 +71,27 @@ QString TransactionInfo::paymentId() const
...
 }
 
+QString TransactionInfo::destinations_formatted() const
+{
+    QString destinations;
+    for (auto const& t: transfers()) {
+        if (!destinations.isEmpty())
+          destinations += "<br> ";
+        destinations +=  WalletManager::instance()->displayAmount(t->amount()) + ": " + t->address();
+    }
```
Comment (moneromooo-monero) [inline src/libwalletqt/TransactionInfo.cpp]: Looks like this leaks the transfers which are newed below

Comment (Jaqueeee) [inline src/libwalletqt/TransactionInfo.cpp]: Fixed. Gets deleted with parent TransactionInfo

File: src/libwalletqt/Transfer.h (hunk 1 above)
Comment (Jaqueeee) [inline src/libwalletqt/Transfer.h]: no. fixed.

File: src/libwalletqt/Transfer.h (hunk 2 above)
Comment (Jaqueeee) [inline src/libwalletqt/Transfer.h]: fixed

**Pull request merged**